src/services/authService.ts

```diff
@@ -0,0 +1,43 @@
+'use server'
+import { Session, SignInDTO } from '@/types/Auth'
+import path from 'path'
+import fs from 'fs'
+import { randomUUID, UUID } from 'crypto'
+import { User } from '@/types/User'
+import { verifyHash, verifySession } from '@/utils/auth'
+
+import { patchUser } from './userService'
+
+export const signIn = async (formData: SignInDTO): Promise<UUID> => {
+    const { login, password } = formData
```

Comment:
Źle Ci podpowiedziałem wcześniej, destrukcje obiektu można wykonać już w samej definicji parametrów funkcji:
```suggestion
export const signIn = async ({ login, password }: SignInDTO): Promise<UUID> => {
```

Reply:
{o,k}